charts/kong/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+### Breaking changes
+
+* The `env.router_flavor` default is now `expressions`. This requires Kong
+  3.4.1 or higher and controller 3.0.0 or higher. Releases using older image
+  versions should change this value to `traditional`
+  [#911](https://github.com/Kong/charts/pull/911)
```

Comment:
Shoulda been marked as draft since it wasn't fully decided--it looks like _only_ `ingress` will get the change.

It's also stuck until the new image is actually out since the gate isn't on otherwise.